libs/dataset-explorer/src/lib/getFeatureBalanceMeasuresChart.ts

```diff
@@ -0,0 +1,209 @@
+    measureLocalization.SorensenDiceCoefficient.Name,
+    {
+      Description: measureLocalization.SorensenDiceCoefficient.Description,
+      KeyName: nameof<IFeatureBalanceMeasure>("SorensonDiceCoeff")
+    }
+  ],
+  [
+    measureLocalization.JaccardIndex.Name,
+    {
+      Description: measureLocalization.JaccardIndex.Description,
+      KeyName: nameof<IFeatureBalanceMeasure>("JaccardIndex")
+    }
+  ],
+  [
+    measureLocalization.KendallRankCorrelation.Name,
+    {
+      Description: measureLocalization.KendallRankCorrelation.Description,
+      KeyName: nameof<IFeatureBalanceMeasure>("KendallRankCorrelation")
+    }
+  ],
+  [
+    measureLocalization.LogLikelihoodRatio.Name,
+    {
+      Description: measureLocalization.LogLikelihoodRatio.Description,
+      KeyName: nameof<IFeatureBalanceMeasure>("LogLikelihoodRatio")
+    }
+  ],
+  [
+    measureLocalization.TTest.Name,
+    {
+      Description: measureLocalization.TTest.Description,
+      KeyName: nameof<IFeatureBalanceMeasure>("TTest")
+    }
+  ],
+  [
+    measureLocalization.TTestPValue.Name,
+    {
+      Description: measureLocalization.TTestPValue.Description,
+      KeyName: nameof<IFeatureBalanceMeasure>("TTestPValue")
+    }
+  ]
+]);
+
+export function getFeatureBalanceMeasuresChart(
+  featureBalanceMeasures: IFeatureBalanceMeasures,
+  selectedFeature: string,
+  selectedMeasure: string
+): IHighchartsConfig {
+  const measureInfo = FeatureBalanceMeasuresMap.get(selectedMeasure);
+  if (
+    featureBalanceMeasures === undefined ||
+    Object.keys(featureBalanceMeasures).length === 0 ||
+    measureInfo === undefined
+  ) {
+    return {};
+  }
+
+  const chartLocalization =
+    localization.ModelAssessment.DataBalance.FeatureBalanceMeasures.Chart;
+
+  const rows = featureBalanceMeasures[selectedFeature];
+  const uniqueClasses = _.uniq(
+    rows.map((r) => r.ClassA).concat(rows.map((r) => r.ClassB))
+  );
+
+  const items: number[][] = [];
+  uniqueClasses.forEach((classA, colIndex) => {
+    uniqueClasses.forEach((classB, rowIndex) => {
+      let row = rows.find((r) => r.ClassA === classA && r.ClassB === classB);
+      let foundFlipped = false;
+
+      if (row === undefined) {
+        // If a row doesn't exist for ClassA vs. ClassB, try to find one for
+        // ClassB vs. ClassA
+        row = rows.find((r) => r.ClassA === classB && r.ClassB === classA);
+        foundFlipped = true;
+      }
+
+      const measureName = measureInfo.KeyName;
+      if (
+        row !== undefined &&
+        measureName !== undefined &&
+        measureName in row
+      ) {
+        // If a measure value is found for ClassB vs. ClassA, flip its value
+        // since every feature balance measure gap is symmetric
+        const measureValue = row[measureName] * (foundFlipped ? -1 : 1);
+        items.push([rowIndex, colIndex, measureValue]);
+      }
+    });
+  });
+
+  // Chosen color scale is "viridis", the default color scale for matplotlib, because it is:
```

Comment:
Does this mesh well with the color scheme we use in AML? For model overview I was asked to use blue tones.

Reply:
@PIC123 Do you have any thoughts on the color scheme? Do you know if AzureML would be okay with my chosen color scheme?

Also, since we are adding this as a flight, how about we keep it to viridis, see if AzureML + users like it, and then make the choice?

Reply:
I think this is fine for now